navbar/Menu.jsx

```diff
@@ -1,34 +1,39 @@
-import { Link } from 'react-router-dom';
-
+import { Link } from "react-router-dom";
 
 const Menu = () => {
-    return(
-       
-
-        
-       <div>
-
-<img className="logo" src="../src/images/Logo.png" alt="" />
+  return (
+    <div>
+      <img className="logo" src="../src/images/Logo.png" alt="" />
 
-        <ul className="nav-list">
+      <ul className="nav-list">
         <li className="cashbook">
-          <Link to='/income' className="nav-item">Kassenbuch</Link>
+          <Link to="/income" className="nav-item">
+            Kassenbuch
+          </Link>
         </li>
         <li className="shopping">
-          <Link to='/shop' className="nav-item">Einkaufsliste</Link>
+          <Link to="/shop" className="nav-item">
```

Comment:
This is just a personal idea, why not using `shopping-list` instead of `shop`? When I read `shop` I'm not thinking of an `shopping-list`. ;-)